src/pages/index.js

```diff
@@ -148,7 +148,7 @@ const renderCard = (item) => {
   );
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**